apps/consumer-client/src/pages/examples/gpc-proof.tsx

```diff
@@ -277,6 +314,16 @@ async function verifyProof(
     return { valid: false, err: "Watermark does not match." };
   }
 
+  const sameMembershipLists = _.isEqual(
```

Comment:
If we were being more strict you'd probably want to compare these as Sets rather than arrays, since order doesn't matter.
This makes me wonder whether membership lists should be sorted by the GPC compiler so that order doesn't affect the circuit inputs.  I'm not sure, though, and certainly not something which has to change in this PR.